### DISCUSSION TITLE
[FR]: If I use dataview in each file to query sth as an inline property, can these info be displayed in db-table? will it become time consuming?

### DISCUSSION THREAD

**Discussion by ghweili** [RafaelGB/obsidian-db-folder#884] [Q&A]
### Present your request

Hi, I have around 10,000 notes which I plan to use db-folder to display.  Morever, I may consider using dataview in each of these notes to query some data from other notes, which may be stored as an inline property in the form of sth:: queried_data or simply as plain text in the main body. 

Since the said query may take a little while for each file. So I'm wondering:

1. whether these queried data can be displayed in db-folder if they are specified as inline property format?
2. if it is possible, will it take a long time to render db-folder page, if these data are to be displayed in db-folder?
3. if it is possible, will it take long if I choose not to display these fields, even though they are stored as inline property?
4. if it is not possible to be queried by db-folder, I guess it is more reasonable to leave them in the main body of the note. In that case, will it take a long time to render db-folder page, which means, will these queries have to be excuted if I use db-folder to collect some of their inline properties? If so, is there any way to avoid the execution of these queries?

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment]
hello @ghweili ,
Your use case could be an interesting case to test the performance of the plugin.
I will response your dots:
* dbfolder supports inline fields without problem since every cell renders obsidian markdown
* the render performance depends of the rows per page, but with pagination theorically the table engine supports 100k rows (https://github.com/tanstack/table)
* about to sort the result of those queries could be possible with a formula but could be computationally expensive
* The filters are one of our core features and you can create a quick filter button to enable/disable them

**Comment by ghweili** [comment]
In  that case,  let's say that we have 10k files, each containing some dataview queries,  if we are going to display these files in db folder, all the dataview queries have to be executed in order to render the db-folder table, regardless of whether the dataview queries are placed inside inline fields or not? 

If so, I guess it would time consuming if the query itself is to be conducted on a relatively large vault, containing, let's say 20k files. Then, one should be cautious about using dataview queries in files that I want to display in db-folder.

**Comment by ghweili** [comment]
please be advised that I also hope to be able to use access the same vault on my two-year-old android mobile phone if possible, with 12+3 memory and an 8-core 2.84Ghz process (Qualcomm Snapdragon 865)

**Reply by RafaelGB** [reply]
I am devoting less time this year for work reasons to the project, so the mobile version is not being maintained.

With the cases that you raise in a computer with a ram between the average you will not have problems
